runtime/tests/runtime_integration_tests/test_pcr.rs

```diff
@@ -0,0 +1,203 @@
+// Licensed under the Apache-2.0 license
+
```

Comment:
Can you add all new tests to test-coverage.md? And also add any test gaps that may have come up due to this PR?

Reply:
What do you mean with 'test gaps'?

Reply:
This doc covers tests in runtime firmware. If there's any features that get added here but are not yet tested, you can add an item to the "Test Gaps" section at the bottom. But if not that's fine (I still need to do a pass to check if I see anything).